perf/index.js

```diff
@@ -0,0 +1,47 @@
+const Benchmark = require('benchmark');
+const fc = require('../lib/fast-check');
+
+const benchSettings = function(label, arb) {

```

Comment:
what does `arb`mean? Maybe you could use the complete word to make code more simple to understand

Reply:
`arb` stands for `arbitrary`, the whole documentation uses this denomination.